roofit/roofit/src/RooJohnson.cxx

```diff
@@ -13,31 +13,70 @@
  * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
  *****************************************************************************/
 
-/** \class RooJohnson
-    \ingroup Roofit
-
-Johnson's \f$ S_{U} \f$ distribution.
-
-This PDF results from transforming a normally distributed variable \f$ x \f$ to this form:
-\f[
-  z = \gamma + \delta \sinh^{-1}\left( \frac{x - \mu}{\lambda} \right)
-\f]
-The resulting PDF is
-\f[
-  \mathrm{PDF}[\mathrm{Johnson}\ S_U] = \frac{\delta}{\lambda\sqrt{2\pi}}
-  \frac{1}{\sqrt{1 + \left( \frac{x-\mu}{\lambda} \right)^2}}
-  \;\exp\left[-\frac{1}{2} \left(\gamma + \delta \sinh^{-1}\left(\frac{x-\mu}{\lambda}\right) \right)^2\right].
-\f]
-
-It is often used to fit a mass difference for charm decays, and therefore the variable \f$ x \f$ is called
-"mass" in the implementation. A mass threshold allows to set the PDF to zero to the left of the threshold.
-
-###References:
-Johnson, N. L. (1949). *Systems of Frequency Curves Generated by Methods of Translation*. Biometrika **36(1/2)**, 149–176. [doi:10.2307/2332539](https://doi.org/10.2307%2F2332539)
-
-\image html RooJohnson_plot.png
```

Comment:
Why not use this plot anymore? I liked it, because it nicely illustrated the shape of the RooJohnson for different values of the parameter in one canvas:
https://root.cern/doc/v632/classRooJohnson.html

I think even with your added parameter studies, it's still a nice complementary plot.